Alcatraz/Controllers/ATZPluginWindowController.m

```diff
@@ -209,7 +209,9 @@ - (void)installPackage:(ATZPackage *)package andUpdateControl:(ATZFillableButton
         [button setFillRatio:progress animated:YES];
     } completion:^(NSError *failure) {
         [ATZStyleKit updateButton:button forPackageState:package animated:YES];
-        if (package.requiresRestart) {
+        if (failure) {
+            [self presentError:failure];
```

Comment:
Not too fond of the modal dialog for network errors, but I guess this is better than just logging.
👍  for now.